src/left-nav-title.json

```
@@ -1,26 +1,86 @@
+    "/docs/test-management/imports-and-exports/zephyr/": "Import Test Data from Zephyr"
+  },
+  "testrail": {
+    "/docs/test-management/imports-and-exports/testrail/": "Import Test Data from Testrail"
+  },
+  "browserstack": {
+    "/docs/test-management/imports-and-exports/browserstack/": "Import Test Data from Browserstack"
+  },
+  "export-tests": {
+    "/docs/test-management/imports-and-exports/export-tests/": "Export Test Data"
+  },
+  "step-groups": {
```

Biome (1.9.4)

[error] The key step-groups was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)